products/knownprojects/python/__init__.py

```diff
@@ -11,14 +11,18 @@
 # Make `dcpy` available
 sys.path.append(str(_proj_root))
 
-S3_SOURCE_BUCKET = "edm-private"
+S3_BUCKET = "edm-private"
 S3_SOURCE_HOUSING_TEAM_DIR = "dcp_housing_team/db-knownprojects"
+S3_OUTPUT_DIR = "db-kpdb"
 
 DATA_PATH = _product_path / "data"
 RAW_DATA_PATH = DATA_PATH / "raw"
 PROCESSED_DATA_PATH = DATA_PATH / "processed"
 CORRECTIONS_DATA_PATH = DATA_PATH / "corrections"
 
+CURRENT_DIR = os.getcwd()
+OUTPUT_DIR = f"{CURRENT_DIR}/output"
```

Comment:
just curious, do we actually want these to change based on where you run python? (as opposed to using one of the fixed paths, e.g. `_product_path / "output"`

Reply:
per this and other comments, simplified/dropped a lot of unused old stuff!